.github/workflows/Sandpit_exs.yml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Install dependency
         run: |
-          mamba install -c conda-forge -c openbiosim/label/main biosimspace python=3.8 ambertools gromacs "sire=2023.2.2" alchemlyb pytest "openff-interchange<0.3.0" "rdkit<2023"
+          mamba install -c conda-forge -c openbiosim/label/main biosimspace python=3.8 ambertools gromacs "sire=2023.2.2" alchemlyb pytest pyarrow openff-interchange "rdkit<2023"
```

Comment:
Why do you need `pyarrow`?

Reply:
For pandas serialisation. Do you not like pyarrow?

Reply:
I don't really have an opinion, was just wondering why you don't use `pickle` or a `csv`?

Reply:
csv won't preserve the index. Pickle is fine until something changes and you are screwed. parquet via pyarrow is the pandas recommended serialisation method, which shall preserve the index as it is.